addons/website/static/tests/tours/configurator_translation.js

```diff
@@ -0,0 +1,62 @@
+/** @odoo-module */
+
+import tour from 'web_tour.tour';
+
+tour.register('configurator_translation', {
+    test: true,
+    url: '/website/configurator',
+},
+[
+    // Configurator first screen
+    {
+        content: "click next",
+        trigger: 'button.o_configurator_show',
+    },
+    // Description screen
+    {
+        content: "select a website type",
+        trigger: 'a.o_change_website_type',
+    }, {
+        content: "insert a website industry",
+        trigger: '.o_configurator_industry input',
+        run: 'text ab',
+    }, {
+        content: "select a website industry from the autocomplete",
+        trigger: '.o_configurator_industry ul li a:contains(in fr)',
+        extra_trigger: 'html[lang*=fr]',
+    }, {
+        content: "select an objective",
+        trigger: '.o_configurator_purpose_dd a',
+    }, {
+        content: "choose from the objective list",
+        trigger: 'a.o_change_website_purpose',
+    },
+    // Palette screen
+    {
+        content: "chose a palette card",
+        trigger: '.palette_card',
+    },
+    // Features screen
+    {
+        content: "select confidentialité",
+        trigger: '.card:contains(confidentialité)',
+    }, {
+        content: "Click on build my website",
+        trigger: 'button.btn-primary',
+    }, {
+        content: "Loader should be shown",
+        trigger: '.o_theme_install_loader_container',
+        run: function () {}, // it's a check
+    }, {
+        content: "Wait untill the configurator is finished",
+        trigger: 'body.editor_started',
+        timeout: 30000,
+    }, {
+        content: "exit edit mode",
+        trigger: '.o_we_website_top_actions button.btn-primary:contains("Annuler")',
```

Comment:
You should simply target the `button[data-action=save]` that will never change, instead of its text content.
Or even better, use the tour util `clickOnSave` :wink:

Reply:
That's what I usually do when I design tours, however, since one of the goal of this commit is also to have [parts of the website translated](https://github.com/odoo/odoo/commit/0eaa4c0ece9927c80c5e2e1991d49093a8f774cc#diff-5e92e473fa4d3da6db7ef727fb217dad51ef6c2383913edca73fe040a23e82c2R427), I thought it would be a good idea to check the language in which the editor is started in (which should be the website's default language). It's something quite hard to test (as not every snippet of every theme is translated and the configurator has a random element to it.) and this step probably should have had a comment explaining why I was using explicit content here.

Should I just drop the idea and exit the editor normally ?

Reply:
(I didn't know about the tour util clickOnSave however, it will be quite useful 😁  )

Reply:
Ok I see, then it is fine to keep "Annuler" or "Sauvegarder", just need to add a quick js comment on top of the trigger step to explain it as it is a bit unusual :)
```js
// Keep the check on french translation `Annuler` as it is the purpose of the test and this step, ensure the editor is displayed in french.
```
Something like that, feel free to rephrase it ;-)